sn_node_manager/src/cmd/node.rs

```diff
@@ -28,18 +31,20 @@ use sn_releases::{ReleaseType, SafeReleaseRepoActions};
 use sn_service_management::{
     control::{ServiceControl, ServiceController},
     rpc::RpcClient,
-    NodeRegistry, NodeService, ServiceStateActions, ServiceStatus, UpgradeOptions, UpgradeResult,
+    NatDetectionStatus, NodeRegistry, NodeService, ServiceStateActions, ServiceStatus,
+    UpgradeOptions, UpgradeResult,
 };
 use sn_transfers::HotWallet;
 use std::{cmp::Ordering, io::Write, net::Ipv4Addr, path::PathBuf, str::FromStr};
 use tracing::debug;
 
 /// Returns the added service names
 pub async fn add(
+    auto_set_nat_flags: bool,
     count: Option<u16>,
     data_dir_path: Option<PathBuf>,
     env_variables: Option<Vec<(String, String)>>,
-    home_network: bool,
+    mut home_network: bool,
```

Comment:
No, the caller does not need to declare the variable mutable. The variable is moved entirely to the function, so we can just make it mutable at the function level.
Instead of having another line which does `let mut home_network = home_network`